### DISCUSSION TITLE
Why doesn't the "paths" object support description/summary

### DISCUSSION THREAD

**Discussion by kentbulza** [OAI/OpenAPI-Specification#3532] [Enhancements]
This seems like an obvious gap to me, but I want to see what were the philosophical reasons for it. Perhaps because the webhooks object is newer, it just wasn't as obvious at first that the info object can be awkward? The issue is there may be very different global(esque) descriptive text for all the paths vs. all the webhooks.

**Comment by charjr** [comment]
I don't know if there's an official reason, but for what it's worth:

I think, because a Paths object contains a bunch of paths (which can have almost any name preceded by a forward slash) no matter what you name this "descriptive text" it could be confusingly similar to someone's path.

For example:
```yaml
paths:
  info:
    
  /info:
```
Or:
```yaml
paths:
  summary:

  \summary:
```

Again, I'm not saying this is the reason. But this would be my guess.

As a possible workaround:
[The Paths Object may be extended with Specification Extensions](https://github.com/OAI/OpenAPI-Specification/blob/3.1.0/versions/3.1.0.md#specification-extensions)

[Which you can find here](
https://github.com/OAI/OpenAPI-Specification/blob/3.1.0/versions/3.1.0.md#specification-extensions)

If you needed it, you could add an `x-description` field with whatever you wanted in it

**Comment by handrews** [comment]
I suspect that this partially because of what you mentioned (webhooks were added later, adding ambiguity to the Info Object), and partially because it's not possible to reference a Paths Object, so it will _always_ appear next to the mandatory Info Object.

I've filed issue #3722 to track the request of the fields, so I'm going to close out this discussion.
